lang/cs/Org.Apache.REEF.IO/FileSystem/AzureBlob/AzureCloudBlobClient.cs

```diff
@@ -62,6 +63,11 @@ public ICloudBlobContainer GetContainerReference(string containerName)
             return new AzureCloudBlobContainer(_client.GetContainerReference(containerName));
         }
 
+        public bool CreateContainerIfNotExists(string containerName)
+        {
+            return _client.GetContainerReference(containerName).CreateIfNotExists();
+        }
```

Comment:
Now that it's in the container please remove.

Reply:
removed